File: C++/CodePracticeInC++/Frequency.cpp
#include <iostream>
using namespace std;

int main() {
    int a[] = {1, 2, 2, 3, 1, 4, 5};
    int n = 7;

    for (int i = 0; i < n; i++) {
        int count = 0;

        // check if element was already counted
        bool alreadyCounted = false;
        for (int k = 0; k < i; k++) {
            if (a[k] == a[i]) {
                alreadyCounted = true;
                break;
            }
        }
        if (alreadyCounted) continue;

        // count frequency
        for (int j = 0; j < n; j++) {
            if (a[j] == a[i]) count++;
        }

        cout << a[i] << " occurs " << count << " times" << endl;
    }
}

File: Java work/Code Practice/Frequency.java
public class Frequency{
    public static void main(String[]args){
        int []a={1,2,2,3,4,1,5,5,2};
        int visited[]=new int[a.length];
        int x=0;
        for(int i=0;i<a.length;i++){
            if (visited[i] == 1)
                continue;

            int count=1;
            for(int j=i+1;j<a.length;j++){
                if(a[i]==a[j]){
                    count++;
                    visited[j]=1;
                   


                }
            }
             System.out.println(a[i]+" "+count);
        }

    }
}